packages/terra-overlay/tests/nightwatch/Overlay/DarkOverlay.jsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 
 import Overlay from '../../../lib/Overlay';
+import OverlayContainer from '../../../lib/OverlayContainer';
 
 const OverlayExample = () => (
-  <Overlay isOpen backgroundStyle={Overlay.Opts.BackgroundStyles.DARK} id="dark-overlay" >
-    <h1>Dark Overlay</h1>
-  </Overlay>
+  <OverlayContainer style={{ height: '100%', width: '100%' }}>
```

Comment:
Why the addition of this?

Reply:
The webdriver io tests were passing only if the Overlay was wrapped within the Overlay container. So added the overlay container.

Reply:
What was the errors you were receiving? These shouldn't be needed.

Reply:
![error](https://user-images.githubusercontent.com/35706779/36047820-cdd47d82-0da2-11e8-85e8-d3945673b506.png)

I was getting this error, I could not replicate this issue with other component wdio tests or even with the overlay tests which have the container wrapper. Also in the [READ Me](https://github.com/cerner/terra-core/tree/master/packages/terra-overlay/docs) of the overlay component the overlay was wrapped within the Overlay container.

Reply:
Ah gottchya, yeah that makes since because the webdriver tests attempt to grab the [data-reactroot] dom element to take a screenshot of. With these screenshot tests, you are attempting to take a screenshot of an element that has height=0. To get a passing test without chancing these test examples, you should update your webdriver.io tests to use the overlay's ID to select for the screenshot:
```
Terra.should.matchScreenshot({ selector: '#light-overlay' });
Terra.should.themeEachCustomProperty('#light-overlay', {
   '--terra-overlay-dark-background-color': 'blue',
   '--terra-overlay-dark-background-image': 'linear-gradient(blue, green)',
   '--terra-overlay-dark-content-color': 'red',
   '--terra-overlay-dark-opacity': '0.8',
});
```

Reply:
Ah okay thanks! Updated the code with the following changes. 
```
  Terra.should.beAccessible();
    Terra.should.matchScreenshot({ selector: '#dark-overlay' });
    Terra.should.themeEachCustomProperty(
      '#dark-overlay',
      {
        '--terra-overlay-dark-background-color': 'blue',
        '--terra-overlay-dark-background-image': 'linear-gradient(blue, green)',
        '--terra-overlay-dark-content-color': 'red',
        '--terra-overlay-dark-opacity': '0.8',
      },
```
And removed the changes made in the other tests.